### ISSUE TITLE
Extract the access token away.

### ISSUE THREAD

**Issue by FooBarQuaxx** [omartawba1/simple_shop#1]
It should be obvious that secrets such as api access tokens should be a matter of configuration. Hardcoding this kind of stuff is not an option.

**Comment by omartawba1** [comment]
@FooBarQuaxx Yes you're right, but i left them for a quick test and a quick start like i mentioned here
https://github.com/omartawba1/simple_shop#important-notice
But you're welcome for any contributions from your end 👍
